testregistry.textproto

```diff
@@ -1001,6 +1001,10 @@ test {
   description: "MPLS in UDP Encapsulation with QoS scheduler"
   readme: ""
 }
+test: {
```

Comment:
For my information, why is this test being tagged under the "TE-*" bucket? Is "TE" basically being overloaded in FNT for "anything programmed through gRIBI"? Note that this is _not_ how "TE" is interpreted internally.

Would it make sense to more explicitly and clearly demarcate the tests based on the project(s) and/or initiatives that own the corresponding feature?

Reply:
Thanks for the feedback, you're correct ,I assigned "TE" (traffic engineering) because this was a gRIBI focused test.  But there is no traffic engineering in this test and while there is a strong gRIBI dependency in this test, its focus is application of QoS with gRIBI.   

I'll move this one to the "DP" (data plane) test id series since it's primary focus is on QoS.